### PR TITLE
Monster descriptions are now loaded from binary

### PR DESCRIPTION
**Pull request by CarlosFdez**

### CODE REVIEW

**Comment by Randomdude1001** [comment]
So, happy to help here:
You are absoluty right, in the german language we can break words into the next line be connect the word parts with a "-" like you saw on your screenshot: "Delika-
tesse"

This means that your solution with the regex is right. The only thing I noticed is that now 2 entries containg "- " (- followed by a space char) are left in the database (id 6 and 12) . The words are Tziki- Ya-Ku and Wildturm- Ödnis. Both words are supposed to be written with -, but we never have a trailing space charackter after a "-".
To make a perfect solution you should replace the sequence "- " with just a single "-" after(!) you put back together the words that were broken up into 2 lines with a "-".

So I´m basically saying that you are right in your assumption, your propossed solution should be right, too. There are just 2 edgecases which could be handled.

Please let me know if you don´t unterstand what I´m trying to say or if you need further assistance.

**Comment by Randomdude1001** [comment]
OK, firstly I have to correct myself concerning:

> but we never have a trailing space charackter after a "-".

there are in fact exceptions like "- und" or "- oder".

Now regarding your changes: All typos I corrected manually before in monster_text are now good. I didn´t even were aware that were the same problem in the item_texts.
I just looked over these (but mainly the german translations) and they look good, too. There are 5 entries left containing "- " from which 3 falls into the exception case "- und" (so all good there) and 2 which should be corrected.

To sum up: It looks even better now and those few edgecases left should be justifiable.

**Comment by CarlosFdez** [comment]
Hmm, how would you handle that use case specifically? replace all -und with - und and -oder with - oder? Or is that only around line breaks that it should do that? Is und/oder with special characters or exactly as written?

**Comment by CarlosFdez** [comment]
Merging, it can always be fixed later.

**Comment by Randomdude1001** [comment]
It´s `WORD- und WORD` respectively `WORD- oder WORD`. So this could be a pattern which could be kind of white-listed.

The 5 entries in `item_text` where the sequence `- ` is present after your changes are:
`select * from item_text where lang_id ='de' and description like '%- %';`
The entires 343, 473, 480 are valid because of `WORD- und/oder WORD`. The entries 532 and 533 are incorrect and the sequence `- ` should simply be deleted.

I´m sure even if you would handle the special german case there are a some edgecases left.

**Comment by CarlosFdez** [comment]
Hmm, those cases you mentioned don't seem to be a case of an invalid line join, I think that's how it is in the game itself. Most of the line break stuff was to to merge the 4 lines of text in the game into a single line, which sometimes has special line join rules as we've just seen. However I think 532 and 533 (Watcher Lens and Nova Brave Trophy) are typos that happen outside of the line breaks.

TBH not sure what the correct choice is regarding in-game typos, but I am leaning towards keeping in game typos unless they're debilitating.
